public/components/stub-modal/stub-modal.js

```diff
@@ -37,7 +37,13 @@ function StubModalInstanceCtrl($rootScope, $scope, $modalInstance, $window, conf
     $scope.sections = getSectionsList(sections);
     $scope.statuses = statusLabels;
     $scope.cdesks = _wfConfig.commissioningDesks;
-    $scope.atomTypes = ['Media'];
+    $scope.atomTypes = _wfConfig.atomTypes.map(type => {
+      if (type === 'storyquestions') {
+        return {value: type, displayName: 'reader questions'};
+      } else {
+        return {value: type, displayName: type};
+      }
+    });
```

Comment:
Could you move this logic into a function to make it a bit more obvious why it's happening ? Maybe `makeAtomTypeDropdownData` or something. Also maybe reader questions should be capitalised?

Reply:
Yes, that sounds sensible. I'll capitalise media atoms as well.